### ISSUE TITLE
[1.8] Complexity on field

### ISSUE THREAD

**Issue by esjee** [rmosolgo/graphql-ruby#1294]
field.rb has some documentation surrounding the complexity, and how to specify a complexity per field: https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/field.rb#L86,L107 

However, it appears that the example doesn't actually work. When attempting to provide complexity either as a keyword argument, or as part of the block, an exception is raised

```
  field :complexField, Integer, complexity: 25
```

```
ArgumentError: unknown keyword: complexity
```

```
  field :complexField, Integer do
        complexity ->(_ctx, _args, _child_complexity) { 25 }
  end
```
```
NoMethodError: undefined method `complexity' for #<GraphQL::Schema::Field:0x00007f3d551849a0>
```

**Comment by esjee** [comment]
I'll try to write a PR to fix this issue. :crossed_fingers:

**Comment by rmosolgo** [comment]
Fixed by #1297
